### DISCUSSION TITLE
Extension Settings: customised from Folder

### DISCUSSION THREAD

**Discussion by Artem0ne** [thunderclient/thunder-client-support#1324] [General]
Hallo @rangav 

i have question regarding TC- Extensions: 
We are working via git on same project. My colleague has this tab "tc_evaluation" in Settings:

![image](https://github.com/rangav/thunder-client-support/assets/59470703/cadf2316-72b3-4d41-98fc-ace6fb382c2b)

However i dont have it. Ic have only standards: **User** and **Workspace**. What settings i need to change to see this Tab? Any Ideas?

Thank you!

**Comment by rangav** [comment]
Hi @Artem0ne 

You need to use `Workspace` tab.

- For Multi-root workspaces use the `Folder` tab

**Reply by Artem0ne** [reply]
Thank you for your fast reply. What i have to enter into this textbox? 
![image](https://github.com/rangav/thunder-client-support/assets/59470703/6d8a50a3-ca0c-4f07-8ba1-5e4ec0c8109e)

**Reply by rangav** [reply]
You should not enter any value in multi root workspace setting, the extension will set the value automatically . Do not modify this value.

**Reply by rangav** [reply]
Also this setting is not used by CLI
